Parsers/ASimAuditEvent/Parsers/ASimAuditEventCrowdStrikeFalcon.yaml

```diff
@@ -0,0 +1,157 @@
+      "update_rule", "Update Rule", "Set", "", "Rule", "Policy Rule",
+      "create_rule", "Create Rule", "Create", "", "Rule", "Policy Rule",
+      "disable_policy", "Disable Policy", "Disable", "", "Policy", "Policy Rule",
+      "delete_policy", "Delete Policy", "Delete", "", "Policy", "Policy Rule",
+      "update_priority", "Update Priority", "Set", "", "Policy", "Policy Rule",
+      "assign_policy", "Assign Policy", "Other", "Assign", "Policy", "Policy Rule",
+      "remove_policy", "Remove Policy", "Other", "Remove", "Policy", "Policy Rule",
+      "ip_rules_added", "IP Rules Added", "Create", "", "Rule", "Other",
+      "ip_rules_removed", "IP Rules Removed", "Delete", "", "Rule", "Other",
+      "CreateAPIClient", "Create API Client", "Create", "", "API Client", "Service",
+      "UpdateAPIClient", "Update API Client", "Set", "", "API Client", "Service"
+  ];
+  let EventSeverityLookup = datatable (LogSeverity: string, EventSeverity: string)
+      [
+      "0", "Informational",
+      "1", "Informational",
+      "2", "Low",
+      "3", "Medium",
+      "4", "High",
+      "5", "High"
+  ];
+  let UserAuditActivities = dynamic(["delete_report_execution", "delete_scheduled_report", "update_scheduled_report", "create_scheduled_report", "update_class_action", "update_policy", "enable_policy", "create_policy", "remove_rule_group", "create_rule_group", "delete_rule_group", "add_rule_group", "delete_rule", "update_rule", "create_rule", "disable_policy", "delete_policy", "update_priority", "assign_policy", "remove_policy", "ip_rules_added", "ip_rules_removed"]);
+  let AuthAuditActivities = dynamic(["CreateAPIClient", "UpdateAPIClient"]);
+  let parser = (disabled: bool=false) {
+      let allData = CommonSecurityLog
+          | where not(disabled)
+          | where (DeviceVendor == "CrowdStrike" and DeviceProduct == "FalconHost")
+          | where (DeviceEventClassID == "UserActivityAuditEvent" and Activity in (UserAuditActivities)) or (DeviceEventCategory == "AuthActivityAuditEvent" and Activity in (AuthAuditActivities))
+          | lookup EventFieldsLookup on Activity
+          | lookup EventSeverityLookup on LogSeverity
+          | extend EventType = EventType_lookup;
+      let useractivitydata = allData
+          | where (DeviceEventClassID == "UserActivityAuditEvent" and Activity in (UserAuditActivities)) 
+          | extend
+              EventStartTime = unixtime_milliseconds_todatetime(tolong(ReceiptTime))
+          | project-rename EventOriginalType = DeviceEventClassID;
+      let authactivitydata = allData
+          | where (DeviceEventCategory == "AuthActivityAuditEvent" and Activity in (AuthAuditActivities))
+          | extend
+              EventStartTime = todatetime(DeviceCustomDate1)
+          | project-rename
+              EventOriginalType  = DeviceEventCategory;
+      union useractivitydata, authactivitydata
+      | extend
+          EventResult = iff(EventOutcome == "false", "Failure", "Success"),
```

Comment:
Hope you have tested with an actual log? I couldn't find EventOutcome in sample

Reply:
In our environment, we are not getting EventOutcome for UserActivityAuditEvent but it's available in config file, so it's possible to have in others. Hence, we have added a condition based on that.